src/transformime/mathwidget.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  Message
+} from 'phosphor-messaging';
+
+import {
+  Widget
+} from 'phosphor-widget';
+
+import {
+  typeset
+} from '../utils/latex';
+
+
+/**
+ * A widget which typesets itself after it is attached.  
+ */
+export
+class MathWidget extends Widget {
```

Comment:
This seems like a weird name for a widget that renders generic html.  Maybe make these two separate widgets, one for Latex and one for HTML?